app/app/models/term.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Boolean, Column, Date, ForeignKey, Integer, String
+
+from app.db.base_class import Base
+from app.models.year import Year
+
+
+class Term(Base):
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(100), index=True)
+    year_id = Column(Integer, ForeignKey(f'{Year.__table__.name}.id', ondelete='CASCADE'), index=True, nullable=False)
+    current_year_term = Column(Integer)
```

Comment:
Shouldn't this be set `nullable=False`? Considering it's required in TermCreate schema.